src/SuperDumpService/Program.cs

```diff
@@ -5,7 +5,7 @@ namespace SuperDumpService {
 	public static class Program {
 		public static void Main(string[] args) {
 			var host = new WebHostBuilder()
-				.UseKestrel()
+				.UseKestrel(opt => opt.Limits.MaxRequestBodySize = 8589934592L)  // 8gb
```

Comment:
at this point, neither the configuration is initialized nor has the Startup method been called. I don't think we can shift loading the config to Main().